### DISCUSSION TITLE
Always show tick value.

### DISCUSSION THREAD

**Discussion by Gungrave223** [plouc/nivo#1617] [Q&A] [answered]
For the line graph with a point scale...is it possible to always show the y-axis tick value regardless of whether or not the data intersect with the value?

And example would be if the x,y dataset have the potential values of 1 through 4....I want is have the y-axis to always have 4 ticks regardless of the actual dataset.

**Comment by wyze** [comment, marked as answer]
You can use the `tickValues` of the `axisLeft` prop. I just checked and the `point` scale doesn't support a min/max property, so it will work if you leave it the default `linear` scale.

```ts
import { Line } from '@nivo/line'

function LineChart() {
  return <Line {...} axisLeft={{ tickValues: [1, 2, 3, 4] }} yScale={{ min: 1, max: 4, type: 'linear' }} />
}
```

CodeSandbox: https://codesandbox.io/s/quirky-knuth-v4bbj

**Reply by Gungrave223** [reply]
@wyze That was exactly what I was looking for...thanks a lot 😀
